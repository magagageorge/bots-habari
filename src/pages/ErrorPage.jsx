import React from "react";

class ErrorPage extends React.Component {
    render() {
        return (<div className="container-fluid animated fadeIn">
            <div className="row mx-0 w-100">
                <div className="col-12 d-flex-justify-content-center">
                    <h4> Page Not Found</h4>
                </div>
            </div>
        </div>)
    }
}

export default ErrorPage;