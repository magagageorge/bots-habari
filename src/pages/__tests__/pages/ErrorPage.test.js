
// Auto-generated do not edit


/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import ErrorPage from '../../ErrorPage';


describe('pages/ErrorPage test', () => {
  it('pages/ErrorPage should match snapshot', () => {
    const component = renderer.create(<ErrorPage
       />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
