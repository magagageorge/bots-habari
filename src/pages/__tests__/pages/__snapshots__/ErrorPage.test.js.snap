// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/ErrorPage test pages/ErrorPage should match snapshot 1`] = `
<div
  className="container-fluid animated fadeIn"
>
  <div
    className="row mx-0 w-100"
  >
    <div
      className="col-12 d-flex-justify-content-center"
    >
      <h4>
         Page Not Found
      </h4>
    </div>
  </div>
</div>
`;
